/**:
  ros__parameters:
    image_topic: /event_camera/image_raw
    depth_topic: /aps_depth
    model_type: MiDaS_small
    device: cpu
    invert_depth: false
    scale: 1.0
    min_depth: 0.001
    max_depth: 100.0
    normalize_depth: true
    norm_low_pct: 5.0
    norm_high_pct: 95.0
    log_stats: true
